### PR TITLE
ENH: improve order of tasks

### LINKED ISSUE #149: Badly needed overhaul of the tear-down section
We should change the order of events in the session tear-down section to avoid going back and forth between the scanning room and outside

EDITED LATER ON BY @oesteban:

In addition to:
- [ ] Revise the order of tasks in session tear-down

originally posted by @esavary, I would add:

- [ ] Make terminology consistent with previous sections (e.g., what is *PC Tower*? What is the *CIBM EEG lab*?)
- [ ] Remove weird statements (put the keys beneath Nora's desk??) OR, if they are still necessary, we should mention Nora's desk somewhere in the beginning (and conceal the personal name as a secret).

### PR DESCRIPTION
**Pull request by esavary**
- Change the order of tasks to parallelize work inside and outside the scanning room. 
- Make terminology consistent.
- Fix some typos and weird statements.
closes #149

### CODE REVIEW

**Comment by esavary** [comment]
I left the mirror task in the 'inside scanning room section' due to its need for special care but moved some other tasks to parallelize the work.

File: docs/data-collection/tear-down.md
Comment (oesteban) [inline docs/data-collection/tear-down.md]: 👏

File: docs/data-collection/tear-down.md
Comment (oesteban) [inline docs/data-collection/tear-down.md]: Too long for a title, let's use the full admonition:
```suggestion
!!! important "The following steps are better done with two people"
    One researcher operates **inside** the Scanning Room and the other one is **outside**, at the Control Room.
    They will parallelize the designated steps below: [inside the scanning room](#after-scan-inside-the-scanner-room), and [outside the scanning room](#after-scan-outside-the-scanner-room).
```

File: docs/data-collection/tear-down.md
Comment (oesteban) [inline docs/data-collection/tear-down.md]:
```suggestion
- [ ] Unplug the two cables (signal and power) connected to the ET arm.
    Put those extremities aside far from the scanner.
```

File: docs/data-collection/tear-down.md
Comment (oesteban) [inline docs/data-collection/tear-down.md]:
```suggestion
```

File: docs/data-collection/tear-down.md
Comment (acionca) [inline docs/data-collection/tear-down.md]: Below (original line 88), there is an instruction to "Put back the regular Lens" - should it be added here as well ?

**Review by acionca — COMMENTED** [review]
I do not know about the different lenses for the ET so my suggestion may be wrong.

File: docs/data-collection/tear-down.md
Comment (esavary) [reply docs/data-collection/tear-down.md]: I think we actually stopped putting the original lens back on the ET, possibly to ease the storage. Maybe @celprov or @ bfranceschiello can confirm this?

Comment (esavary) [reply docs/data-collection/tear-down.md]: I just had a confirmation from @ bfranceschiello , we do not need to put it back.

Comment (acionca) [reply docs/data-collection/tear-down.md]: Oh alright, my bad, thanks for the clarification :)

**Comment by esavary** [comment]
No problem, it's a good point! It was the occasion to check it was not something we forgot during the last sessions.Le 30 août 2023 18:17, Alexandre Cionca ***@***.***> a écrit :
@acionca commented on this pull request.



In docs/data-collection/tear-down.md:
>  
-- [ ] Switch off the projector.
+- [ ] Help the person inside the scanning room to extract the cables and tubes.
+- [ ] Carefully roll the fiber and power cable of the ET, and place them in the rolling table of the ET PC Tower.
+- [ ] Carefully roll RB and the GA tubes and put them back in the GA and BIOPAC boxes.
+- [ ] Unscrew the ET lens, while **ALWAYS** keeping one hand under the lens while screwing/unscrewing it and put it back into its cover.

Oh alright, my bad, thanks for the clarification :)

File: docs/data-collection/tear-down.md
Comment (celprov) [inline docs/data-collection/tear-down.md]:
```suggestion
    The home (:fontawesome-solid-house:) button can alternatively be used to bring the bed out.
```

File: docs/data-collection/tear-down.md
Comment (celprov) [inline docs/data-collection/tear-down.md]:
```suggestion
    - [ ] Carefully extract the four cables one by one (the fiber and power of the ET, the tube of RB and the tube of the GA) back through the access tube. 
```

File: docs/data-collection/tear-down.md
Comment (celprov) [inline docs/data-collection/tear-down.md]: Is there a reason why the RB and GA tubes as distinguished from the ET cables? We used to remove everything at the same time through the same hole, so might as well get concatenate in one sentence.
```suggestion
```

File: docs/data-collection/tear-down.md
Comment (celprov) [inline docs/data-collection/tear-down.md]: Because the GA position has changed, not relevant anymore
```suggestion
```

File: docs/data-collection/tear-down.md
Comment (celprov) [inline docs/data-collection/tear-down.md]: @esavary Have you tested this on our setup? From what I remember, when I exited the Eyelink application, I was left with a black screen, so "click on the Shutdown button from the File Manager toolbar" would not have been possible.

File: docs/data-collection/tear-down.md
Comment (celprov) [inline docs/data-collection/tear-down.md]:
```suggestion
- [ ] Switch off the laptop. Plug back the sync box and the VGA projector where they were.
```

File: docs/data-collection/tear-down.md
Comment (celprov) [inline docs/data-collection/tear-down.md]:
```suggestion
- [ ] Bring back the ET PC Tower and the plexiglas in {{ secrets.rooms.projector | default("███") }}.
```

**Review by celprov — DISMISSED** [review]
Looks good :) Please address the question regarding how to turn off the ET PC tower.

**Comment by oesteban** [comment]
I plan to also work on this today, please do not merge until I'm done.

File: docs/data-collection/tear-down.md
Comment (esavary) [reply docs/data-collection/tear-down.md]: I will double-check the EyeTracker. The last time I tried to exit the application, I had a DOS-like screen.

**Comment by oesteban** [comment]
@esavary you can accept comments in batch so that we do not need to squeeze before merging. To do so, you'll need to go to the "Files changed" tab and "Add suggestion to batch".

File: docs/data-collection/tear-down.md
Comment (oesteban) [inline docs/data-collection/tear-down.md]: This needs more coordination (like for the communication to check the tightness of the RB)
```suggestion
- [ ] Agree with another experimenter, who will exit the room, sit on the other side of the access tubes in the Control Room, and pull cables (fiber and power of the ET) and tubes (RB and GA) out of the room.
    At the same time, you carefully feed them so they do not suffer abrasion from touching the edges of the tube.
```

File: docs/data-collection/tear-down.md
Comment (oesteban) [inline docs/data-collection/tear-down.md]: This comment also replaces the following four lines (36-39 of the original text), GitHub does not allow me to modify them directly.

```suggestion
- [ ] The experimenter currently outside in the Control Room will enter the Scanner Room with the plastic bag of the mirror and a fresh pair of gloves and will leave them prepared on the bed.
- [ ] Remove the mirror frame from its rails mounted on the head coil and lay it on the bed.
- [ ] Put the gloves on and detach the infrared mirror from the standard mirror.
- [ ] Immediately insert the infrared mirror back into its plastic pouch.
- [ ] Take the pouch containing the infrared mirror outside to the Control Room and place it back in its designated box, with extreme care.
- [ ] Clean the standard mirror removing all residues of glue from the scotch tape.
- [ ] Re-attach the mirror to the coil's frame.
```

File: docs/data-collection/tear-down.md
Comment (oesteban) [inline docs/data-collection/tear-down.md]: Several things are repated or suggested later, but should be here:

After line 22
```
- [ ] Prepare the belt on the bed to be removed from the room when you show the participant out.
```

After line 23 (after adding the above line, 24)
```
- [ ] Disconnect the last section of the cannula and dispose of it in the trash can.
```

File: docs/data-collection/tear-down.md
Comment (oesteban) [inline docs/data-collection/tear-down.md]: This is repeated, done already in ~22-24
```suggestion
```

File: docs/data-collection/tear-down.md
Comment (oesteban) [inline docs/data-collection/tear-down.md]: The cannula tube has already been removed at this point
```suggestion
- [ ] Disconnect the ECG cable from the filter of the access panel, roll the cable.
- [ ] Take the ET arm and the ECG cable outside to the Control Room.
- [ ] Place the ET arm in a stable place.
- [ ] Store the ECG cable in its designated box / storage.
- [ ] Store the ET arm in its designated box.
```

**Review by oesteban — COMMENTED** [review]
My first pass. Apologies for the delay

File: docs/data-collection/tear-down.md
Comment (oesteban) [inline docs/data-collection/tear-down.md]: @esavary I haven't been able to interpret this. Is this what we are doing? What did AcqKnowlege have to do with this?

Once this item is clear, we are ready to merge this one.

File: docs/data-collection/tear-down.md
Comment (esavary) [inline docs/data-collection/tear-down.md]:
```suggestion
### Readying the scanner for the next session
```

File: docs/data-collection/tear-down.md
Comment (esavary) [reply docs/data-collection/tear-down.md]: I removed lines 113-115 since they were no longer relevant. I am considering moving the transfer to Dropbox under "data management".WDYT?

Comment (oesteban) [reply docs/data-collection/tear-down.md]: I'd be more confident if we make sure the files are uploaded to Dropbox on the spot.

We should change the psychopy experiments to write directly on the shared folder instead of in the git repo.

Comment (oesteban) [reply docs/data-collection/tear-down.md]: See TheAxonLab/HCPh-fMRI-tasks#24

File: docs/data-collection/tear-down.md
Comment (oesteban) [reply docs/data-collection/tear-down.md]: Since this comment has not been resolved within this PR, I just opened #166 to follow up.

**Review by oesteban — APPROVED** [review]
I have opened issues for the few things that remain unresolved after this PR

**Pull request merged**
